pkg/scheduler/schedule_one.go

```diff
@@ -576,21 +576,20 @@ func (sched *Scheduler) findNodesThatPassFilters(
 
 // numFeasibleNodesToFind returns the number of feasible nodes that once found, the scheduler stops
 // its search for more feasible nodes.
-func (sched *Scheduler) numFeasibleNodesToFind(numAllNodes int32) (numNodes int32) {
-	if numAllNodes < minFeasibleNodesToFind || sched.percentageOfNodesToScore >= 100 {
+func numFeasibleNodesToFind(percentageOfNodesToScore, numAllNodes int32) (numNodes int32) {
+	if numAllNodes < minFeasibleNodesToFind || percentageOfNodesToScore >= 100 {
```

Comment:
`percentageOfNodesToScore == 100` is enough, we have validated this must <= 100.

Reply:
Good point. Remove it.